x/incentives/keeper/genesis_test.go

```diff
@@ -9,12 +9,80 @@ import (
 	"github.com/stretchr/testify/require"
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
 
+	"github.com/osmosis-labs/osmosis/osmomath"
 	osmoapp "github.com/osmosis-labs/osmosis/v19/app"
 
+	"github.com/osmosis-labs/osmosis/v19/x/concentrated-liquidity/model"
 	"github.com/osmosis-labs/osmosis/v19/x/incentives/types"
 	lockuptypes "github.com/osmosis-labs/osmosis/v19/x/lockup/types"
 )
 
+var (
+	distrToByDuration = lockuptypes.QueryCondition{
+		LockQueryType: lockuptypes.ByDuration,
+		Denom:         "lptoken",
+		Duration:      time.Second,
+	}
+
+	distrToNoLock = lockuptypes.QueryCondition{
+		LockQueryType: lockuptypes.NoLock,
+	}
+
+	distrToNoLockPool1 = lockuptypes.QueryCondition{
+		LockQueryType: lockuptypes.NoLock,
+		Denom:         "no-lock/i/1",
+		Duration:      time.Hour * 24 * 7,
+		Timestamp:     time.Time{}.UTC(),
+	}
+
+	distrToNoLockPool2 = lockuptypes.QueryCondition{
+		LockQueryType: lockuptypes.NoLock,
+		Denom:         "no-lock/i/2",
+		Duration:      time.Hour * 24 * 7,
+		Timestamp:     time.Time{}.UTC(),
+	}
+
+	distrToByGroup = lockuptypes.QueryCondition{LockQueryType: lockuptypes.ByGroup}
+
+	gaugeCoins = sdk.Coins{sdk.NewInt64Coin("stake", 10000)}
+
+	gaugeOneRecord = types.InternalGaugeRecord{
+		GaugeId:          1,
+		CurrentWeight:    osmomath.NewInt(100),
+		CumulativeWeight: osmomath.NewInt(200),
+	}
+
+	gaugeTwoRecord = types.InternalGaugeRecord{
+		GaugeId:          2,
+		CurrentWeight:    osmomath.NewInt(100),
+		CumulativeWeight: osmomath.NewInt(200),
+	}
+
+	expectedGroups = []types.Group{
+		{
+			GroupGaugeId: 5,
+			InternalGaugeInfo: types.InternalGaugeInfo{
+				TotalWeight:  gaugeOneRecord.CurrentWeight.Add(gaugeTwoRecord.CurrentWeight),
+				GaugeRecords: []types.InternalGaugeRecord{gaugeOneRecord, gaugeTwoRecord},
+			},
+			SplittingPolicy: types.ByVolume,
+		},
+	}
+
+	expectedGroupGauges = []types.Gauge{
+		{
+			Id:                5,
+			IsPerpetual:       true,
+			DistributeTo:      distrToByGroup,
+			Coins:             sdk.Coins(nil),
+			NumEpochsPaidOver: 0,
+			FilledEpochs:      0,
+			DistributedCoins:  sdk.Coins(nil),
+			StartTime:         time.Time{}.UTC(),
+		},
+	}
```

Comment:
I think we might have this defined in another file as well

Reply:
I searched the codebase for an predefined gauge array with an Id equal to 5 and did not find it

Reply:
Can we put this gauge in the slice:
https://github.com/osmosis-labs/osmosis/blob/2c2893a2df3d5ab23149a26ab1fd90ba0a993ffb/x/incentives/keeper/distribute_test.go#L67-L74

If changed need to be made to gauge, we can use these helpers to make mutations:
https://github.com/osmosis-labs/osmosis/blob/2c2893a2df3d5ab23149a26ab1fd90ba0a993ffb/x/incentives/keeper/distribute_test.go#L1219-L1349

Reply:
I am probably misunderstanding, but if we use this defaultPerpetual gauge we will need to
1. Create a copy
2. Change the distributeTo field
3. Change the coins field
4. Change the start time field
5. Change the filled epochs field
6. Change the distributed coins field

This feels like it would be more confusing and more lines than creating a new variable